queue-worker/main.go

```diff
@@ -176,6 +176,8 @@ func postResult(client *http.Client, req queue.Request, result []byte, statusCod
 	}
 
 	request, err := http.NewRequest("POST", req.CallbackURL.String(), reader)
+	request.Header.Set("Content-Type", req.Header.Get("Content-Type"))
```

Comment:
I actually wonder if this is enough. Perhaps we should be setting _all_ of the headers that came from the responding function.